src/patterns/instance-role.ts

```diff
@@ -10,7 +10,8 @@ import {
   GuSSMRunCommandPolicy,
 } from "../constructs/iam";
 
-interface InstanceRoleProps extends GuGetS3ObjectPolicyProps, Partial<GuLogShippingPolicyProps> {
+interface InstanceRoleProps extends GuGetS3ObjectPolicyProps {
+  withoutLogShipping?: boolean; // optional to have log shipping added by default, you have to opt out
```

Comment:
We might want to start using https://typedoc.org/ to generate a documentation site and also provide hints via IDEs IntelliSense features.